testbed/tests/test_window.py

```diff
@@ -487,6 +488,11 @@ async def test_as_image(main_window, main_window_probe):
     main_window_probe.assert_image_size(screenshot.size, main_window_probe.content_size)
 
 
+async def test_screen(main_window, main_window_probe):
+    assert isinstance(main_window.screen, ScreenInterface)
+    main_window_probe.assert_screen_implementation_type(main_window.screen)
```

Comment:
Again - we don't care about the type; we care about the functionality.

In this case, it's a little difficult to test, because we can't guarantee that the test device will have multiple screens. The best option I can think of would be to validate absolute and relative position on every *available* screen. The test will be slightly different on a multiple-screen machine, but there's not much we can do to work around that.

Reply:
Done

Reply:
Once again... *what* have you done? I don't see any change.

Reply:
These changes:

https://github.com/proneon267/toga/blob/1b8a52ddbd091800567da5b92b3e1e5c0e8ea8ee/testbed/tests/test_window.py#L491-L506